agent/agent.go

```diff
@@ -1045,6 +1050,8 @@ func (a *agent) handleReconnectingPTY(ctx context.Context, logger slog.Logger, m
 				read, err := rpty.ptty.OutputReader().Read(buffer)
 				if err != nil {
 					// When the PTY is closed, this is triggered.
+					// Error is typically a benign EOF, so only log for debugging.
+					logger.Debug(ctx, "PTY output read error", slog.Error(err))
```

Comment:
```suggestion
					logger.Debug(ctx, "unable to read pty output, command exited?", slog.Error(err))
```
Since this is expected to be benign, let's make it sound less bad 😄